q2_quality_control/_threshold_graph/_visualizer.py

```diff
@@ -41,15 +43,48 @@ def _check_inputs(**kwargs):
 TEMPLATES = pkg_resources.resource_filename(
     'q2_quality_control._threshold_graph', 'assets')
 
+#generates seqeunce table and fasta files for each assignment
+def write_table_fastas(output_dir, dest, display_sequences, sequences, df, desig, decontam_scores, read_nums, table):
+    _blast_url_template = ("http://www.ncbi.nlm.nih.gov/BLAST/Blast.cgi?"
+                           "ALIGNMENT_VIEW=Pairwise&PROGRAM=blastn&DATABASE"
+                           "=nt&CMD=Put&QUERY=%s")
+    with open(os.path.join(output_dir, dest), 'w') as fh:
+        for index, row in df.iterrows():
+            fh.write(format_fasta(row['id'], row['Sequence']))
```

Comment:
See my earlier comment about not implementing this yourself, or write unit tests for `format_fasta`

Reply:
format_fasta function removed in favor  of the data being read in as a DNAIterator

addressed in [adfe12a](https://github.com/qiime2/q2-quality-control/pull/82/commits/adfe12ad4213c9923face4267028d18902427b8a)